torchci/components/LogViewer.tsx

```diff
@@ -194,13 +194,22 @@ function Log({ url, line }: { url: string; line: number | null }) {
 export default function LogViewer({
   job,
   logRating = LogAnnotation.NULL,
-  showAnnotationToggle = false,
+  showAnnotationToggle = process.env.DEBUG_LOG_CLASSIFIER === "true",
+  maxNumFailureLines = process.env.DEBUG_LOG_CLASSIFIER === "true" ? 2 : 1,
 }: {
   job: JobData;
   logRating?: LogAnnotation;
   showAnnotationToggle?: boolean;
+  maxNumFailureLines?: number;
 }) {
-  const [showLogViewer, setShowLogViewer] = useState(false);
+  // const numFailureLines =
```

Comment:
Same here, are these lines commented out for debugging?

Reply:
This one specifically is not.